integration-tests/rest-plugin.js

```diff
@@ -161,6 +175,89 @@ describe('REST API Plugin', function () {
 
     });
 
+    describe('#Plugin List', function () {
+        before(function (done) {
+            if (!utils.pluginUrl) {
+                this.skip();
+            }
+
+            function createPlugin(callback) {
+
+                var params = {
+                    jwt: jwtWithPermissions,
+                    data: {
+                        name: PLUGIN1,
+                        description: description,
+                        parameters: {
+                            jsonString: paramObject
+                        }
+                    }
+                };
+
+                params.query = path.query(
+                    'deviceId', DEVICE_ID,
+                    'networkIds', networkId,
+                    'names', '',
+                    'returnCommands', true,
+                    'returnUpdatedCommands', true,
+                    'returnNotifications', true
+                );
+
+                utils.createPlugin(path.current, params, function (err, result) {
+                    callback();
+                })
+            }
+
+            async.series([
+                createPlugin
+            ], done);
+        });
+
+        it('should get all user plugins', function (done) {
+            utils.getPlugin(path.current, {jwt: jwtWithPermissions}, function (err, result) {
+                assert.strictEqual(!(!err), false, 'No error');
+                console.log(result);
+                assert.strictEqual(utils.core.isArrayOfLength(result, 1), true, 'Is array of 1 object');
+                utils.matches(result[0], {
+                    name: PLUGIN1
+                });
+
+                done();
+            })
+        });
+
+        it('should get all plugins with admin token', function (done) {
+            utils.getPlugin(path.current, {jwt: utils.jwt.admin}, function (err, result) {
+                assert.strictEqual(!(!err), false, 'No error');
+                console.log(result);
+                assert.strictEqual(result.length > 0, true);
+
+                done();
+            })
+        });
+
+        it('should fail with 403 on list all plugins', function (done) {
```

Comment:
Specify test name (e.g. 'should fail with 403 on list all plugins for invalid token' or 'should fail with 403 on list all plugins for token without permission')